stacks-signer/src/client/stackerdb.rs

```diff
@@ -26,97 +27,87 @@ use libstackerdb::{StackerDBChunkAckData, StackerDBChunkData};
 use slog::{slog_debug, slog_warn};
 use stacks_common::codec::{read_next, StacksMessageCodec};
 use stacks_common::consts::SIGNER_SLOTS_PER_USER;
-use stacks_common::types::chainstate::{StacksAddress, StacksPrivateKey, StacksPublicKey};
+use stacks_common::types::chainstate::StacksPrivateKey;
 use stacks_common::{debug, warn};
 
 use super::ClientError;
 use crate::client::retry_with_exponential_backoff;
-use crate::config::Config;
+use crate::config::SignerConfig;
 
 /// The StackerDB client for communicating with the .signers contract
 pub struct StackerDB {
-    /// The stacker-db session for the signer StackerDB.  Used for querying signer addresses and
-    /// other system metadata.
-    signers_stackerdb_session: StackerDBSession,
     /// The stacker-db sessions for each signer set and message type.
-    /// Maps (signer-set, message ID) to the DB session.
-    signers_message_stackerdb_sessions: HashMap<(u32, u32), StackerDBSession>,
+    /// Maps message ID to the DB session.
+    signers_message_stackerdb_sessions: HashMap<u32, StackerDBSession>,
     /// The private key used in all stacks node communications
     stacks_private_key: StacksPrivateKey,
     /// A map of a (signer-set, message ID) to last chunk version for each session
-    slot_versions: HashMap<(u32, u32), HashMap<u32, u32>>,
-    /// The signer ID -- the index into the signer list for this signer daemon's signing key.
+    slot_versions: HashMap<u32, HashMap<u32, u32>>,
```

Comment:
The comment for this might be out-of-date -- you're only mapping the message ID to the (slot ID, slot version) pair now, right?